src/core_plugins/kbn_vislib_vis_types/public/controls/point_series/category_axis.js

```diff
@@ -0,0 +1,33 @@
+import _ from 'lodash';
+import uiModules from 'ui/modules';
+import vislibValueAxesTemplate from 'plugins/kbn_vislib_vis_types/controls/point_series/category_axis.html';
+const module = uiModules.get('kibana');
+
+module.directive('vislibCategoryAxis', function ($parse, $compile) {
```

Comment:
consider removing unused imports $parse, $compile. Similar to other point_series/* directives.